purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -43,8 +43,7 @@ open class PurchasesConfiguration(builder: Builder) {
             this.service = service
         }
 
-        // TODO: make public
-        internal fun store(store: Store) = apply {
+        fun store(store: Store) = apply {
```

Comment:
do we want this to be public? since it defaults to PLAY and the AmazonConfiguration sets it to AMAZON? is there another use case i'm not thinking of?

Reply:
so initially I didn't include the api key as required so devs could do things like:

```
        val builder = PurchasesConfiguration.Builder(this)
        if (BuildConfig.STORE.equals("amazon")) {
            builder.store(Store.AMAZON).apiKey("public_amazon_sdk_key")
        } else if (BuildConfig.STORE.equals("google")) {
            builder.apiKey("public_google_sdk_key")
        }
        Purchases.configure(builder.build())
```

But then I realized that it was better to make the api key mandatory in the Builder constructor just in case someone forgets to set it, which means that code would look like:

```
        var builder: PurchasesConfiguration.Builder = null
        if (BuildConfig.STORE.equals("amazon")) {
            builder = PurchasesConfiguration.Builder(this, "public_amazon_sdk_key").store(Store.AMAZON)
        } else if (BuildConfig.STORE.equals("google")) {
            builder = PurchasesConfiguration.Builder(this, "public_google_sdk_key")
        }
        Purchases.configure(builder.build())
```

Which is true, kinda defeats the point of having the store public, but I don't think it hurts right?